bme_test_suite.yaml

```diff
@@ -0,0 +1,37 @@
+---
+- hosts: deploy_node
+  name: Install and run tests
+  vars:
+    tempest_root: /opt/tempest_untagged
+    test_name: persistent-resources-tests
+    test_type: persistent-clean
+
+  tasks:
+    - name: "Clone git repo {{ test_name }} into {{ tempest_root }}/{{ test_name }}"
+      when: test_name == "persistent-resources-tests"
+      command: /usr/local/bin/ansible utility_all -i /opt/openstack-ansible/playbooks/inventory -m shell -a "git clone https://github.com/osic/{{ test_name }}.git {{ tempest_root }}/{{ test_name }}"
+
+    - name: "Install {{ test_name }} into {{ tempest_root }}/{{ test_name }}"
+      when: test_name != "tempest"
+      command: /usr/local/bin/ansible utility_all -i /opt/openstack-ansible/playbooks/inventory -m shell -a "pip install {{ tempest_root}}/{{ test_name }}"
```

Comment:
Other playbook is cruft at this point, but it hasn't been removed yet as we haven't yet validated that this playbook will cover all the other tests as well. That will likely be removed once we get to that stage of the tests and we've tested this works as expected.